docs/DesignDocs/Attestation_API_Proposal.md

```diff
@@ -159,15 +160,19 @@ generation and verification in
 manner.
 - Remove legacy artifacts that have become redundant, e.g. the SGX
 local / remote flags.
+- Incorporate changes proposed in [PR #3089](https://github.com/openenclave/openenclave/pull/3089).
 
 Proposed Headers files:
 
-- Attester public API to be declared in `openenclave/attestation/attester.h`
-- Verifier public API to be declared in `openenclave/attestation/verifier.h`
+- Attester public API to be declared in `<openenclave/attestation/attester.h>`
+- Verifier public API to be declared in `<openenclave/attestation/verifier.h>`
+- TEE-agnostic claims IDs to be declared in `<openenclave/bits/evidence.h>`
```

Comment:
```suggestion
- TEE-agnostic claim IDs to be declared in `<openenclave/bits/evidence.h>`
```

Also, evidence.h is fine for now but in the future, we might want claim ids (or at least some subset of them) to be reusable by attestation results too, in which case we may want a header like claims.h or something that can be included by both evidence.h and whatever is used for attestation results.

Reply:
Agreed there is further clean up as the attestation API definition evolves. This header file `<openenclave/bits/evidence.h>` is existing, and this PR tried to avoid changes not directly to the proposed features.